src/sentry/api/endpoints/project_details.py

```diff
@@ -621,99 +617,108 @@ def put(self, request: Request, project) -> Response:
                 ]
         # TODO(dcramer): rewrite options to use standard API config
         if has_project_write:
-            options = request.data.get("options", {})
-            if "sentry:origins" in options:
+            request_options = request.data.get("options", {})
```

Comment:
please remove this unnecessary change, and try to refactor it separately.

Reply:
This change is necessary because of shadowing, should I just change the name of the import from `options` to `sentry_options`?

Reply:
oh i see, yeah probobly `from sentry import options as sentry_options`